.github/workflows/acceptance.yml

```diff
@@ -304,29 +304,19 @@ jobs:
         uses: ./.github/actions/artifacts
         if: ${{ needs.files-changed.outputs.backend == 'true' }}
 
-  visual-diff:
-    # This guarantees that we will only schedule Visual Snapshots if all
-    # workflows that generate artifacts succeed
+  # If this job fails then visual-diff will not be scheduled
+  acceptance-required-check:
     needs: [acceptance, frontend, chartcuterie]
+    # XXX: Cannot rename this check since it is now required
     name: triggers visual snapshot
     # This is necessary since a failed/skipped dependent job would cause this job to be skipped
     if: always()
     runs-on: ubuntu-20.04
     timeout-minutes: 20
 
     steps:
-      # If any jobs we depend on fail, we will fail since this checks triggers Visual Snapshots which is a required check
+      # If any jobs we depend on fails, this job will fail since visual-diff.yml depends on this succeeding
       - name: Check for failures
         if: contains(needs.*.result, 'failure')
         run: |
           echo "One of the dependent jobs have failed. You may need to re-run it." && exit 1
-
-      - name: Diff snapshots
-        id: visual-snapshots-diff
-        uses: getsentry/action-visual-snapshot@v2
-        # Do not execute on forks or on master. Forks are handled in visual-diff.yml
-        if: github.head_repository.full_name == 'getsentry/sentry' && github.ref != 'refs/heads/master'
```

Comment:
As indicated in my commit message, this step has been skipped over the last week, however, `visual-diff.yml` from `master` has been scheduling the Visual Snapshots.